tests/integration/test_flow_integration.py

```diff
@@ -33,7 +33,7 @@ def validate_callback(resp):
         )
 
 
-def test_clip_batch():
+def test_with_batch():
```

Comment:
do not use `validate_callback` directly in the test.

Reply:
Struggling to understand, why not use it in the test directly? Should it be defined outside the test?

Reply:
u need to use mock or return. 

The reason is, what if the callback is not called due to some error in the Flow. Then the test would pass but it should not

Reply:
Ah, got it, thanks :)

Reply:
Changed to return results https://github.com/jina-ai/executor-image-torch-encoder/pull/1/commits/74622f9f55919b6c6ba008a9383aebdc554909c7